Dockerfile

```
@@ -3,6 +3,7 @@
 #---------------------------------------------------------------------------------------------
 
 # Build with builder image to reduce image size
+ARG APE_VERSION
```

Just to give it a more descriptive name
```suggestion
ARG BASE_APE_IMAGE_TAG
```

---

Also, should this be where the default goes?

---

Typically, you default the way I have it, but if you want verbosity, and for it to be very clear, I can change this